README.md

```diff
@@ -0,0 +1,43 @@
+#  📸 Unsplash X Pexels MoodBoard 🎥
+
+A website by the Lally Enthusiasts
+
+## Project Description
+
+This repository is the front end of a full-stack web application using the tools and techniques we've learned over the semester, including React, Redux, Router, PostgreSQL, Sequelize, Express, and Node.
```

Comment:
its should say the repo is the back end of a full-stack web app

Reply:
Otherwise, great work on building the readme file Kazi!

Reply:
GOOD CATCH GEORGE!! I'll fix it